### DISCUSSION TITLE
419 Page Expired - When creating a rule

### DISCUSSION THREAD

**Discussion by 0Knull** [firefly-iii/firefly-iii#9439] [Firefly III]
Hi. I'm trying to create a rule but when I click 'Store new rule', it immediately goes to the "419 | Page Expired" page. I'm not sure where to begin.

**Comment by JC5** [comment]
Mmm, neither do I. Is it only on this particular page?

**Comment by 0Knull** [comment]
Yes, so far.. I've been to probably most of the available pages and hadn't had an issue yet. I have the importer as well and that's working.

**Comment by 0Knull** [comment]
nevermind, it decided it's working now. I didn't do anything.
